package.json

```diff
@@ -41,4 +41,4 @@
     "gulp-useref": "^2.1.0",
     "rename": "^1.0.4"
   }
-}
```

Comment:
I think we should leave package.json as-is.

Reply:
Agree. I think it causes by the editor settings. I just feel if it does not fail the CI, the PR should be safe for this repo.

Reply:
Right I remove this issue and creating new pull request